Gemfile.lock

```diff
@@ -4,36 +4,36 @@ PATH
     xpm_ruby (0.1.0)
       activesupport
       builder
-      dry-types
+      dry-types (~> 0.15.0)
```

Comment:
A lot of gems upgraded as a result of my `bundle install`, and dry-types is using an older syntax so needs to be locked to this version.

The tests otherwise pass fine with the other upgrades.